packages/babel-plugin-import-jsx-pragma/index.js

```diff
@@ -32,47 +35,77 @@ module.exports = function( babel ) {
 	function getOptions( state ) {
 		if ( ! state._options ) {
 			state._options = Object.assign( {}, DEFAULT_OPTIONS, state.opts );
+			if ( state._options.isDefault && state._options.scopeVariableFrag ) {
+				// eslint-disable-next-line no-console
+				console.warn( 'scopeVariableFrag is only available when isDefault is false' );
+				state._options.scopeVariableFrag = null;
+			}
 		}
 
 		return state._options;
 	}
 
 	return {
 		visitor: {
-			JSXElement( path, state ) {
```

Comment:
Can you explain why this was changed?

Reply:
Yes, I noticed that without this change, the following test fails (added in this PR):

```js
it( 'adds import for scope variable for Fragments', () => {
	const original = 'let foo = <></>;';
	const string = getTransformedCode( original );

	expect( string ).toBe( 'import React from "react";\nlet foo = <></>;' );
} );
```

```
  ● babel-plugin-import-jsx-pragma › adds import for scope variable for Fragments

    expect(received).toBe(expected) // Object.is equality

    Expected: "import React from \"react\";
    let foo = <></>;"
    Received: "let foo = <></>;"

      27 |
      28 |
    > 29 |              expect( string ).toBe( 'import React from "react";\nlet foo = <></>;' );
         |                               ^
      30 |      } );
      31 |
      32 |      it( 'does nothing if there is no jsx', () => {

      at Object.toBe (packages/babel-plugin-import-jsx-pragma/test/index.js:29:20)
```

It seems that `JSX` is an alias for most or all JSX that may be encountered.

It seems that `<></>` should cause the import to be added.